### DISCUSSION TITLE
PageInfo type already registered

### DISCUSSION THREAD

**Discussion by mouldt** [ChilliCream/hotchocolate#3948] [Q&A]
I have a project where the GraphQL code is split between assemblies.
If I have a Query extension class in two assemblies and both have a resolver that has [UsePaging] i get this error
The name `PageInfo` was already registered by another type. (HotChocolate.Types.Pagination.PageInfoType)
Any ideas why I would get this and how to resolve please?

**Comment by PascalSenn** [comment]
Can you share a repro uf this issue?

**Reply by mouldt** [reply]
whats the preferred way to share?

**Reply by mouldt** [reply]
I now have a test project setup.
I am not sure if this is related but It looks like its the attribute params that cause some issues:
This is in one assembly:
        [UsePaging(MaxPageSize = 10, IncludeTotalCount = true)]
        [UseFiltering]
        public IQueryable<ModelObj> GetapiVersion()

This is in another assembly:
        [UsePaging(IncludeTotalCount = false)] or [UsePaging]
        [UseFiltering]
        public IQueryable<ModelObj> GetAltapiVersion()

The difference in the UsePaging attribute causes this error when requesting the schema:
The name `ModelObjConnection` was already registered by another type. (HotChocolate.Types.Pagination.ConnectionCountType<HotChocolate.Types.ObjectType<ClassLibrary1.ModelObj>>)
